test/Datadog.Trace.ClrProfiler.Native.Tests/clr_helper_test.cpp

```diff
@@ -9,6 +9,8 @@ class CLRHelperTest : public ::CLRHelperTestBase {};
 
 TEST_F(CLRHelperTest, EnumeratesTypeDefs) {
   std::vector<std::wstring> expected_types = {
+      L"Microsoft.CodeAnalysis.EmbeddedAttribute",
```

Comment:
@pjanotti Any idea what required these additional attributes? At first I thought it's GlobalSupressions.cs, but then the solution already references linked solution level GlobalSuporessions. + Something turned on code analysis for dependency samples that wasn't required in previous commit. 

CC - @pellared @dszmigielski

Reply:
By the namespace this is an attribute type added by Roslyn, but I don't know its purpose. Can you point me to the commit in which wasn't required?

Reply:
I think it was the first commit (7bce5a0) then nuke was added (d8b7e40) and additional attributes appeared.

Reply:
@RassK from a quick check that attribute is added to netstandard1.0 and netstandard2.0 assemblies - I'm not sure but perhaps this was not added with some older version of Roslyn. Another possibility is that they were always there but the test didn't error with they were not listed (although the macro seems to imply equality).